### PR TITLE
Releasing 3.0.0

### PR DESCRIPTION
**Pull request by hush-hush**

### CODE REVIEW

**Review by derekwbrown — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+# 3.0.0 / 2019-03-20
+
+- [IMPROVEMENT] Use 'include_tasks' instead of 'include' which bump minimum ansible version to 2.4 (thanks to [@rouge8][]).
+- [FIX] Make sure the Live Process agent and APM agent aren't started when datadog_enabled is set to false (thanks to [@pdecat][]).
```
Comment (olivielpeau) [inline CHANGELOG.md:9]: the links don't work :)

**Comment by rouge8** [comment]
Any plans to finish this release?

**Comment by dabcoder** [comment]
> Any plans to finish this release?

Yes, apologies about the delay, we're planning on finish this very soon, I will update this thread asap.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+# 3.0.0 / 2019-05-17
+
+- [FEATURE] On Unix: you can now add the Agent's user to additionnal groups.
```
Comment (dabcoder) [inline CHANGELOG.md]: Nit: Linux instead of Unix?

**Review by dabcoder — CHANGES_REQUESTED** [review]

**Pull request merged**